roles/idrac_bios/molecule/update_attributes_immediate/cleanup.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Converge file to update attributes with apply time as Immediate
+  hosts: all
+  gather_facts: false
+  vars:
+    idrac_ip: "{{ lookup('env', 'IDRAC_IP') }}"
```

Comment:
No use of vars mentioned in this file, if not needed remove it. Check name too, need update.

Reply:
Removed and updated the file